.github/workflows/build.yml

```diff
@@ -33,11 +33,11 @@ jobs:
       - name: Set up Node
         uses: actions/setup-node@46071b5c7a2e0c34e49c3cb8a0e792e86e18d5ea
         with:
-          node-version: '14'
+          node-version: '16'
 
       - name: Update NPM
         run: |
-          npm install -g npm@7
+          npm install -g npm@8
```

Comment:
Do we still need this?

Reply:
@Hinton: Tested this on a separate branch and it builds fine. Removed it.